dts/bindings/input/gpio-qdec.yaml

```diff
@@ -35,6 +35,18 @@ properties:
     description: |
       GPIO for the A and B encoder signals.
 
+  led-gpios:
+    type: phandle-array
+    description: |
+      GPIOs for LED or other components needed for sensing the AB singnals.
+
+  led-pre-us:
+    type: int
+    default: 5
```

Comment:
defaults need to be justified

Reply:
Thanks, missed that one. Fixed.

Reply:
doesn't seem to be justified :-)

Reply:
Well, explanation on why 5 is the right default and not e.g. 10

Reply:
ok I see what you mean, 5 is somewhat arbitrary and close to the specific board I used to test this with, feels like a reasonable number to me but coming from a sample size of one I certainly won't pretend it's ideal and the actual value should be measured on the board, it depends on the led pin slew rate, led drive current, coupling factor and input pin parasitics plus any other thing in the middle, I added a note saying that it should be measured on the board, honestly any number is as good as any other, I think this has to be tweaked per design. It really should have been required but it's only required if you have the led pins specified and 0 is a valid number.

Reply:
if that's the case, then it should be required, no defaults

Reply:
Yes but it's only required if `led-gpios` is specified and `idle-poll-time-us` is > 0 so I can't just flag it as required. I can default it to -1 and and a build check for the condition above? How does that sound?

Reply:
or maybe no default, and check if node has prop under certain conditions, using a build assert

Reply:
oh, right, forgot I can use HAS_PROP, good idea I'll do that

Reply:
I'm embarrassed to admit how long it took me to write that condition correctly :-) ok dropped the default added the build assert, dropped the default from `idle-poll-time-us` as well and moved it to the code, makes the condition read a bit easier, added a test build entry as well just because there's 10 type of people in the world and seemingly I'm in the wrong one this morning.